bin/rl/rl.cpp

```diff
@@ -228,7 +228,7 @@ TARGETINFO TargetInfo[] = {
    "sh5c",       FALSE, FALSE, FALSE, "sh5sim",       "libcsim.lib", "x86asm",
    "sh5m",       FALSE, FALSE, FALSE, "sh5sim",       "libcsim.lib", "x86asm",
    "ia64",       FALSE, TRUE,  TRUE,  NULL,           NULL, "x86asm",
-   "amd64",      FALSE, TRUE,  TRUE,  NULL,           NULL, "x86asm",
```

Comment:
Agree, we should have both.  Internal tests may not be converted and rl still needs to work with them.

`ia64` is different from `amd64` though, I wouldn't expect those to be aliases.